azure/bootnode-image/bootnode-image-step-1.sh

```diff
@@ -71,7 +71,7 @@ echo $output
 vmip=$(echo $output | jq '.publicIpAddress' | tr -d '"')
 echo "VM IP address: $vmip"
 
```

Comment:
the main branch is replaced with mas810-alpha-shajsyed
undo this change

Reply:
Thanks Amrin , will remove this after my testing is done and my branch is ready for merge